extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcConfig.java

```diff
@@ -10,31 +10,31 @@
 import io.quarkus.runtime.annotations.ConfigRoot;
 import io.smallrye.config.ConfigMapping;
 import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithDefaults;
 import io.smallrye.config.WithParentName;
+import io.smallrye.config.WithUnnamedKey;
 
 @ConfigMapping(prefix = "quarkus.oidc")
 @ConfigRoot(phase = ConfigPhase.RUN_TIME)
 public interface OidcConfig {
 
-    /**
-     * The default tenant.
-     */
-    @WithParentName
-    OidcTenantConfig defaultTenant();
+    String DEFAULT_TENANT_KEY = "<default>";
 
     /**
      * Additional named tenants.
      */
-    @ConfigDocSection
     @ConfigDocMapKey("tenant")
     @WithParentName
+    @WithUnnamedKey(DEFAULT_TENANT_KEY)
+    @WithDefaults
```

Comment:
This means that when someone does `namedTenants("unknown")`, he will receive tenant config with defaults even though noone configured it. It doesn't break anything, so I think it is ok. But I don't understand motivation, do you have some plans with it?

Reply:
Hmm, alright, sorry Sergey, I am bit slow today. Now I get it, you need `@WithDefaults` too because you want to see default tenant values. I withdraw this question again. Please close this comment thread.

Reply:
@michalvavrik Np at all, I don't really know all the mechanics behind it - I copied it from the hibernate-orm config